pkg/issuer/acme/http/pod.go

```diff
@@ -184,3 +199,20 @@ func (s *Solver) buildPod(ch *v1alpha1.Challenge) *corev1.Pod {
 		},
 	}
 }
+
+// Merge object meta from the pod template. Fall back to default values.
+func (s *Solver) mergePodObjectMetaWithPodTemplate(pod *corev1.Pod, podTempl *v1alpha1.ACMEChallengeSolverHTTP01IngressPodTemplate) *corev1.Pod {
+	if podTempl == nil {
+		return pod
+	}
+
+	for k, v := range podTempl.Labels {
+		pod.Labels[k] = v
```

Comment:
Being overly cautious, but it's probably best to initialise these maps if they are nil to begin with. This case should pretty much never come up, but emphasis >>should<< 😓